platform-sdk/docs/proposals/proposal-process.md

```diff
@@ -0,0 +1,119 @@
+# Platform Design Proposal Process
+
+This document describes the process for creating and reviewing platform design proposals.
+
+The status of all active design proposals can be viewed
+at [Platform Design Proposals](https://github.com/orgs/hashgraph/projects/73/views/1) project board.
+
+---
+
+## Design Proposal Process Flow
+
+![](designProposalFlow.svg)
+
+## Creating A Proposal
+
+All proposals have an owner who is responsible for taking it through the process to completion. Proposals must be
+presented in the form of a PR containing the proposal in a subdirectory of `platform-sdk/docs/proposals/`. Proposals may
+contain diagrams and supporting materials. Diagrams must be in SVG format and all content must be in an editable
+format (i.e. no images).
+
+Any source code that was developed as part of the design process may be included in the subdirectory of the proposal
+but not merged into the source code of the platform.
+
+While in this draft phase the proposal may be updated and modified as needed. It is best practice to solicit review from
+stakeholders to surface any potential change requests while the proposal is in the draft phase. All grammar and spelling
+errors should be caught during this phase.
+
+### Creating a Draft Proposal
+
+1. Create an issue in the github repository for the creation of the proposal if it is for a project prioritized by
+   SwirldsLabs. Other proposals do not require a ticket.
+2. Create a new branch for the proposal.
+3. Copy the template `template.md` into a subdirectory of `platform-sdk/docs/proposals/` with the name of the
+   proposal.
+4. Fill out the template with the details of the proposal, removing any irrelevant sections. Diagrams and other files
```

Comment:
Can I ignore parts of the template? Can I add custom sections? How important is it to use the template structure?

Reply:
As stated, any irrelevant sections can be removed. It is important to consider each section of the template and if it applies to your proposal, but it is up to the owner/owning team to determine what makes sense and what does not. If any section is not included and a voter thinks it needs to be there, you may be heading for another round of voting.

I think it's fine to add additional information and sections. I will add this.